ivadomed/loader/loader.py

```diff
@@ -37,6 +38,7 @@ def load_dataset(data_list, bids_path, transforms_params, model_params, target_s
         metadata_type (string): None if no metadata
         dataset_type (string): training, validation or testing
         requires_undo (Bool): If True, the transformations without undo_transform will be discarded
+        soft_input(Bool): if True ground truth will be float32 non binarized images
```

Comment:
```suggestion
        soft_input (Bool): if True ground truth will be float32 non binarized images
```
or remove if in kwargs